app/utils/DateTimeFormats.scala

```diff
@@ -19,19 +19,25 @@ package utils
 import play.api.i18n.Lang
 
 import java.time.format.DateTimeFormatter
+import java.time.{Instant, ZoneId}
 import java.util.Locale
 
 object DateTimeFormats {
 
-  private val dateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
+  private val dateFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy 'at' h:mma (z)").withZone(ZoneId.of("GMT"))
 
-  private val localisedDateTimeFormatters = Map(
-    "en" -> dateTimeFormatter,
-    "cy" -> dateTimeFormatter.withLocale(new Locale("cy"))
+  private val localisedDateFormatters = Map(
+    "en" -> dateFormatter,
+    "cy" -> dateFormatter.withLocale(new Locale("cy"))
   )
 
-  def dateTimeFormat()(implicit lang: Lang): DateTimeFormatter = {
-    localisedDateTimeFormatters.getOrElse(lang.code, dateTimeFormatter)
+  def dateFormat()(implicit lang: Lang): DateTimeFormatter = {
+    localisedDateFormatters.getOrElse(lang.code, dateFormatter)
+  }
+
+  def dateTimeFormat(dateTime: Instant): String = {
+    dateTimeFormatter.format(dateTime).replace("AM","am").replace("PM","pm")
```

Comment:
If you use the formatter you don't need to replace the AM and PM as h:mma already does that.